src/components/TweetComp.tsx

```diff
@@ -0,0 +1,44 @@
+import { Pencil, Trash2 } from 'lucide-react';
+import { FC } from 'react';
+import { Tweet, User } from '../../keelClient';
+import Link from 'next/link';
+import { DeleteButton } from './ui/DeleteButton';
+import { handleDeleteTweet } from '@/app/actions/deleteTweet';
+
+type ITweetComp = {
+	tweet: Tweet;
+	user: User;
+};
```

Comment:
I like to use "I" when defining an interface not a Type so probably change this to an interface, or use a type if you don't plan to extend this.